scripts/fetchai_code_quality/internal/static_analysis.py

```diff
@@ -53,7 +53,7 @@ def read_static_analysis_yaml(project_root, build_root, yaml_file_path):
             file_path = d['FilePath']
             absolute_path = file_path if isabs(
                 file_path) else abspath(join(build_root, file_path))
-            assert isfile(absolute_path)
+            #assert isfile(absolute_path), "Not a file: {}".format(absolute_path)
```

Comment:
presumably this should not be commented out in master.
Either this assert should be here or it shouldn't?

Reply:
This assert is valid and should be restored.

Reply:
Oops.
Soz.